### ISSUE TITLE
cannot update Node debug

### ISSUE THREAD

**Issue by MAXMASTER001** [Microsoft/vscode-node-debug#186]
cannot update Node debug. I think this gievs me the error "Debug adapter process has terminated unexpectedly (exit code: 1)"

![image](https://user-images.githubusercontent.com/11816983/49395763-748e7800-f748-11e8-9c78-287f9085cf21.png)

**Comment by roblourens** [comment]
@sandy081 this shouldn't be marked as updateable

**Comment by sandy081** [comment]
Any extension that exists in the market place is updateable. Why do not you want?

**Comment by roblourens** [comment]
Because it's a builtin and it needs to be locked to the vscode version

**Comment by roblourens** [comment]
@MAXMASTER001 what version of vscode do you have?

How did you update the extension? I can't update it at all.

**Comment by sandy081** [comment]
@roblourens Not sure. May be @weinand can confirm.

**Comment by weinand** [comment]
@sandy081 Built-in extensions (e.g. ms-vscode.node-debug and ms-vscode.node-debug2)  are not updatable. They should have no UI for updating and uninstalling.

Builtin extensions should not appear in the `.vscode/extensions` folder. The only reason they could end-up there is that the user copied them there or installed a *.vsix containing them.

**Comment by sandy081** [comment]
Ok, I double checked in the code we neither show update buttons not auto update built in extensions. Not sure how user got into this. Also user says deleting `.vscode/extensions` folder fixed this, does that mean user has manually downloaded the node debug extension into that folder?

**Comment by roblourens** [comment]
I filed an issue quite a while ago because both Matt and I had the node or node2 extension in our .vscode/extensions folder, and neither of us knew how it got there.

It's possible that we did something to put it there for development at some point but I didn't think that was the case. It's possible that there's some weird 1/10^6 bug that can cause it to get installed there?

**Comment by weinand** [comment]
for development we load the built-in extensions to ´.build/builtInExtensions´ inside the vscode project folder.

**Comment by roblourens** [comment]
Right, "for development" I mean we might have installed a vsix or something but I didn't think that was the case. Anyway I can't find the other issue.

**Comment by sandy081** [comment]
An extension goes to `.vscode/extensions` folder is when you install it from Marketplace or as vsix otherwise not.

**Comment by sandy081** [comment]
Sorry I cannot explain that 😞
